pkg/sdk/logging/model/output/kafka.go

```diff
@@ -147,6 +148,194 @@ type KafkaOutputConfig struct {
+	// Signal that librdkafka will use to quickly terminate on rd_kafka_destroy(). If this signal is not set then there will be a delay before rd_kafka_wait_destroyed() returns true as internal threads are timing out their system calls. If this signal is set however the delay will be minimal. The application should mask this signal as an internal signal handler is installed.
+	InternalTerminationSignal int `json:"internal.termination.signal,omitempty"`
+	// Request broker's supported API versions to adjust functionality to available protocol features. If set to false, or the ApiVersionRequest fails, the fallback version `broker.version.fallback` will be used. **NOTE**: Depends on broker version >=0.10.0. If the request is not supported by (an older) broker the `broker.version.fallback` fallback is used.
+	ApiVersionRequest bool `json:"api.version.request,omitempty"`
+	// Timeout for broker API version requests.
+	ApiVersionRequestTimeoutMs int `json:"api.version.request.timeout.ms,omitempty"`
+	// Dictates how long the `broker.version.fallback` fallback is used in the case the ApiVersionRequest fails.
+	ApiVersionFallbackMs int `json:"api.version.fallback.ms,omitempty"`
+	// Older broker versions (before 0.10.0) provide no way for a client to query for supported protocol features (ApiVersionRequest, see `api.version.request`) making it impossible for the client to know what features it may use. As a workaround a user may set this property to the expected broker version and the client will automatically adjust its feature set accordingly if the ApiVersionRequest fails (or is disabled). The fallback broker version will be used for `api.version.fallback.ms`. Valid values are: 0.9.0, 0.8.2, 0.8.1, 0.8.0. Any other value >= 0.10, such as 0.10.2.1, enables ApiVersionRequests.
+	BrokerVersionFallback string `json:"broker.version.fallback,omitempty"`
+	// Allow automatic topic creation on the broker when subscribing to or assigning non-existent topics. The broker must also be configured with `auto.create.topics.enable=true` for this configuration to take effect. Note: the default value (true) for the producer is different from the default value (false) for the consumer. Further, the consumer default value is different from the Java consumer (true), and this property is not supported by the Java producer. Requires broker version >= 0.11.0.0, for older broker versions only the broker configuration applies.
+	AllowAutoCreateTopics bool `json:"allow.auto.create.topics,omitempty"`
+	// Protocol used to communicate with brokers.
+	SecurityProtocol string `json:"security.protocol,omitempty"`
+	// A cipher suite is a named combination of authentication, encryption, MAC and key exchange algorithm used to negotiate the security settings for a network connection using TLS or SSL network protocol. See manual page for `ciphers(1)` and `SSL_CTX_set_cipher_list(3).
+	SslCipherSuites string `json:"ssl.cipher.suites,omitempty"`
```

Comment:
It's a consistent approach across the Go standard library and most Go codebases to capitalize all letters of an initialism when it's part of an identifier. `Ssl` --> `SSL`
Please change, where applicable.